docs/covering-dates.bnf

```diff
@@ -0,0 +1,27 @@
+<month> ::= "Jan" | "Feb" | "Mar" | "Apr" | "May" | "June" | "July" | "Aug" | "Sept" | "Oct" | "Nov" | "Dec"
```

Comment:
This is the grammer that Adam asked for - I figured it would be a good idea if it were in version control. I'm also happy to delete it if @rwalpole thinks it doesn't add anything.

Reply:
I'm OK with it remaining.